pkgs/shells/zsh/zsh-histdb-fzf/default.nix

```
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchFromGitHub, fzf, zsh-histdb }:
+
+stdenv.mkDerivation rec {
+  pname = "zsh-histdb-fzf";
+  version = src.rev;
```

```suggestion
  version = "unstable-XXXX-XX-XX"
```
replace the placeholders with the date of the commit.